packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -913,7 +922,7 @@ export = {
         // WHEN
         new appmesh.VirtualNode(stack, 'test-node', {
           mesh: sharedMesh,
-        });
+        }),
```

Comment:
What happened here?

Reply:
I must have incorrectly changed this during addressing the merge conflict. Reverted the change.